Makefile

```diff
@@ -5,7 +5,7 @@ help:	## Show all Makefile targets.
 
 format:	## Run code autoformatters (black).
 	black .
-	black docs/examples/* --preview -l 79
+	black docs/examples/* --preview --line-length 79
```

Comment:
@nerdai I think we should use `pre-commit` here as well, for symmetry

Reply:
Yeah, sorry just realized that's what you intended in your suggestion 😅.

Making the change now, but I ran into an issue of trying to run pre-commits by name. Both pre-commit hooks will run if I just use their common id: `pre-commit run black-jupyter`

Reply:
Oh, then common ID is the way to go!  Sorry for misleading on the `name`

Reply:
No problemo! It looks like one can also make use of `alias` perhaps too. I'll just the common ID for this though.

I've just pushed the updated Makefile. Everything should work now. 🙏